src/SecretsApi/SecretsApi.php

```diff
@@ -65,6 +65,7 @@ public function listSecrets(string $site_id, bool $debug = false): array
                 'name' => $secretKey,
                 'type' => $secretValue->Type,
                 'value' => $secretValue->Value,
+                'scopes' => implode(', ', $secretValue->Scopes),
```

Comment:
@kporras07 this needs to be an array, not a comma-separated list of scope. Requirements for CMS-865 were changed recently. Sorry for the inconvenience.

Reply:
hmmm, code is expecting it to be an array coming from the API and it's converting it to comma-separated list just for displaying purposes.

I just tested this code against cms-865 branch and it's working